com.pypyrev.linuxdcpp.json

```diff
@@ -18,25 +18,31 @@
 			"name": "boost",
 			"sources": [{
 					"type": "archive",
-					"url": "https://boostorg.jfrog.io/artifactory/main/release/1.76.0/source/boost_1_76_0.tar.bz2",
-					"sha256": "f0397ba6e982c4450f27bf32a2a83292aba035b827a5623a14636ea583318c41"
+					"url": "https://boostorg.jfrog.io/artifactory/main/release/1.83.0/source/boost_1_83_0.tar.bz2",
+					"sha256": "6478edfe2f3305127cffe8caf73ea0176c53769f4bf1585be237eb30798c3b8e"
 				}],
 			"buildsystem": "simple",
 			"build-commands": [
-				"mv boost $CPLUS_INCLUDE_PATH"
+				"./bootstrap.sh --prefix=${FLATPAK_DEST}",
+				"./b2 -j$FLATPAK_BUILDER_N_JOBS install"
 			],
 			"cleanup": ["/include"]
 		},
 		{
 			"name": "libnotify",
 			"sources": [{
 					"type": "archive",
-					"url": "https://github.com/GNOME/libnotify/archive/refs/tags/0.7.9.tar.gz",
-					"sha256": "9bd4f5fa911d27567e7cc2d2d09d69356c16703c4e8d22c0b49a5c45651f3af0"
+					"url": "https://download.gnome.org/sources/libnotify/0.8/libnotify-0.8.2.tar.xz",
```

Comment:
isn't this API breaking?

Reply:
Yea you're right Thanks

Reply:
Actually, looks like this wasn't API breaking